CRM/Contribute/WorkflowMessage/Contribution/BasicContribution.php

```diff
@@ -158,7 +158,7 @@ private function addExampleData(GenericWorkflowMessage $messageTemplate, $exampl
    */
   private function getNonQuickConfigPriceSet(): ?array {
     // Permission check defaults to true - likely implicitly OK but may need to be false.
-    return PriceSet::get()
+    return PriceSet::get(FALSE)
```

Comment:
the comment above it could go too

Reply:
@demeritcowboy good spotting - I grepped for it & found one more place to change to FALSE